core/commands/bitswap.go

```diff
@@ -16,7 +16,7 @@ import (
 
 var BitswapCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "A set of commands to manipulate the bitswap agent",
+		Tagline:          "A set of commands to manipulate the bitswap agent.",
```

Comment:
@RichardLitt if you grep for the entire `Tagline:`, most of the strings don't have the punctuation.